logger.js

```diff
@@ -56,7 +57,7 @@ function pinoLogger (opts, stream) {
 
 function asReqValue (req) {
   return {
-    id: req.id,
+    id: req[id],
```

Comment:
obv this won't stringify so this will affect workflow with log viewers (i.e. kibana) - we index by time anyway, but seeing the unique id on each requests helps to differentiate in your mind